bundles/org.eclipse.cdt.lsp.clangd/plugin.xml

```diff
@@ -109,6 +109,15 @@
             </visibleWhen>
          </command>
       </menuContribution>
+      <menuContribution
+            allPopups="false"
+            locationURI="popup:org.eclipse.ui.popup.any?after=org.eclipse.lsp4e.openCallHierarchy">
```

Comment:
> `popup:org.eclipse.ui.popup.any` 

without `visibleWhen` specified it may appear in the menu for those objects where it is not expected

Reply:
The *Fine References* entry comes along with the generic text editor as well. It's disabled for a file which are not backed by a LS:
![image](https://github.com/eclipse-cdt/cdt-lsp/assets/123444711/7a167fb2-781b-4590-910e-49b1679e7358)

Reply:
from org.eclipse.ui.genericeditor.plugin.xml:
![image](https://github.com/eclipse-cdt/cdt-lsp/assets/123444711/9535761e-210f-47a1-a71f-b48185dc06f4)

We have to re-add it, because our C/C++ editor has a different editor id and therefor the `org.eclipse.ui.genericeditor.GenericEditor.active` reference returns false.